src/native/minipal/random.h

```diff
@@ -3,8 +3,15 @@
 
 #pragma once
 
-#include "pal_compiler.h"
-#include "pal_types.h"
+#include <libs/Common/pal_compiler.h>
+#ifdef __cplusplus
+extern "C"
+{
+#endif // __cplusplus
 
 PALEXPORT void SystemNative_GetNonCryptographicallySecureRandomBytes(uint8_t* buffer, int32_t bufferLength);
 PALEXPORT int32_t SystemNative_GetCryptographicallySecureRandomBytes(uint8_t* buffer, int32_t bufferLength);
```

Comment:
Instead of pulling the function over as it is in System.Native, we should use signatures that are not specific to libs/System.Native and consistent with the rest of native/minipal naming. For example, something like:

```C
int32_t minipal_get_cryptographically_secure_random_bytes(uint8_t* buffer, int32_t buffer_length);
```

And leave the `SystemNative_*` functions under libs and have them call into the ones here.